connector_amazon/components/backend_adapter.py

```
@@ -0,0 +1,728 @@
+        """
+        # Price feeds are submitted through the generic feed adapter
+        # This is a wrapper for consistency
+        feed_adapter = self.component(usage="feed.adapter")
+        return feed_adapter.create_feed("POST_PRODUCT_PRICING_DATA", feed_content)
+
+
+class AmazonInventoryAdapter(AmazonBaseAdapter):
+    _name = "amz.inventory.adapter"
+    _usage = "inventory.adapter"
+
+    def create_inventory_feed(self, feed_content, marketplace_ids):
```

Ruff (0.15.2)

[warning] Unused method argument: `feed_content`

(ARG002)